### DISCUSSION TITLE
I’m going to try out Zeptoforth

### DISCUSSION THREAD

**Discussion by mschwartz** [tabemann/zeptoforth#109] [General]
I’m /u/mykes on Reddit.  We’ve had quite a few discussions there.

for ~$30 US, I picked up one of these:

https://www.amazon.com/gp/product/B0BK9W4H2Q/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

and one of these:

https://www.amazon.com/gp/product/B0BFB53Y2N/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

I should be good to go by Tuesday July 2nd.

looking forward to it.  This forth looks terrific!

let me know if I need something else or ordered a wrong part.

**Comment by tabemann** [comment]
Sorry I missed your comment! Yes, these will work with zeptoforth perfectly well (and as you bought a Pico W you'll be able to use zeptoIP with it).
